man/J.Rd

```diff
@@ -2,57 +2,58 @@
 \alias{J}
 \alias{CJ}
 \alias{SJ}
-\title{ Creates a Join data table }
+\title{
+  Creates a join \code{data.table}
+}
 \description{
-  Creates a \code{data.table} to be passed in as the \code{i} to a \code{[.data.table} join.
+  Creates a \code{data.table} for use in \code{i} in a \code{[.data.table} join.
 }
 
 \usage{
-# DT[J(\dots)]                           # J() only for use inside DT[\dots].
-SJ(\dots)                                # DT[SJ(\dots)]
-CJ(\dots, sorted = TRUE, unique = FALSE)  # DT[CJ(\dots)]
+DT[J(\dots)]                              # J() only for use inside DT[...].
+SJ(\dots)                                 # DT[SJ(...)]
+CJ(\dots, sorted = TRUE, unique = FALSE)  # DT[CJ(...)]
 }
 
 \arguments{
   \item{\dots}{ Each argument is a vector. Generally each vector is the
-  same length but if they are not then the usual silent repetition is applied. }
-  \item{sorted}{ logical. Should the input be sorted (ascending order)? If \code{FALSE}, the input order is retained. }
+  same length, but if they are not then the usual silent recycling is applied. }
+  \item{sorted}{ logical. Should \code{setkey()} be called on all the columns in the order they were passed to \code{CJ}? }
```

Comment:
phrasing here is a bit off... currently could imply setkey is applied repeatedly in the order of `...`